calico.go

```diff
@@ -125,12 +125,19 @@ func cmdAdd(args *skel.CmdArgs) error {
 				return err
 			}
 
+			// Get labels
+			labels := make(map[string]string)
```

Comment:
They technically do the same thing in this context, right? Trying to understand.